xla/pjrt/c/pjrt_c_api_gpu_test.cc

```diff
@@ -437,13 +435,10 @@ TEST(PJRTGpuDeviceTopologyTest, CreateGpuTopology) {
       reinterpret_cast<const PJRT_TopologyDescription*>(args.topology);
   ASSERT_NE(pjrt_topology, nullptr);
 
-#ifdef TENSORFLOW_USE_ROCM
-  EXPECT_EQ(pjrt_topology->topology->platform_id(), xla::RocmId());
-  EXPECT_EQ(pjrt_topology->topology->platform_name(), xla::RocmName());
-#else
-  EXPECT_EQ(pjrt_topology->topology->platform_id(), xla::CudaId());
-  EXPECT_EQ(pjrt_topology->topology->platform_name(), xla::CudaName());
-#endif
+  EXPECT_THAT(pjrt_topology->topology->platform_id(),
+              testing::AnyOf(xla::CudaId(), xla::RocmId()));
+  EXPECT_THAT(pjrt_topology->topology->platform_name(),
```

Comment:
This is fine so far, but it would be good to test that platform_id and name are consistent (both Cuda, or both Rocm)